src/main/java/Jake.java

```diff
@@ -0,0 +1,78 @@
+import java.util.Scanner;
+
+public class Jake {
+
+    // List out all tasks
+    private static void listTask(Task[] commands, int totalCommands) {
+        System.out.println("Current commands being executed: ");
+        for (int i=0; i<totalCommands; i++){
+            System.out.println(Integer.toString(i+1) + "." + commands[i].getStatus());
+        }
+    }
+
+    // Mark or Unmark respective task.
+    private static void markOrUnmarkTask(Task[] commands, int totalCommands, String userInput) {
+        int taskNumber = Integer.parseInt(userInput.substring(userInput.length()-1));
+        if (taskNumber>totalCommands){
+            System.out.println("Task does not exist!");
+        } else if (userInput.startsWith("unmark")){
+            System.out.println("Successfully unmarked");
+            commands[taskNumber-1].markAsUndone();;
+        } else {
+            System.out.println("Successfully marked");
+            commands[taskNumber-1].markAsDone();
+        }
+    }
+
+    // Add Task. Allows ToDos, Deadlines, and Events
+    private static void addTask(Task[] commands, int totalCommands, String userInput) {
+        Task newTask;
+        if (userInput.startsWith("todo")){
```

Comment:
Very readable code, consider using switch cases to improve readability even more